CHANGELOG.md

```diff
@@ -57,6 +57,12 @@ Swift 5.0
 Swift 4.2
 ---------
 
+* [SE-0214][]:
```

Comment:
You'll need to add a reference to the end of this file. The missing references are:

```
[SE-0213]: <https://github.com/apple/swift-evolution/blob/master/proposals/0213-literal-init-via-coercion.md>
[SE-0214]: <https://github.com/apple/swift-evolution/blob/master/proposals/0214-DictionaryLiteral.md>
[SE-0215]: <https://github.com/apple/swift-evolution/blob/master/proposals/0215-conform-never-to-hashable-and-equatable.md>
[SE-0216]: <https://github.com/apple/swift-evolution/blob/master/proposals/0216-dynamic-callable.md>
```

Reply:
The others are not mention on the document yet so I only added [SE-0213]

Reply:
This changelog entry will probably need to be moved to the Swift 5.0 section.

The other typealias from #17711 could be documented here (Swift 4.2) instead.